core/templates/components/ck-editor-helpers/ck-editor-copy-content-service.spec.ts

```diff
@@ -0,0 +1,200 @@
+// Copyright 2020 The Oppia Authors. All Rights Reserved.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS-IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+/**
+ * @fileoverview Unit tests for the Ck editor copy content service.
+ */
+
+import { CkEditorCopyContentService } from
+  'components/ck-editor-helpers/ck-editor-copy-content-service';
+import { HtmlEscaperService } from 'services/html-escaper.service';
+import { LoggerService } from 'services/contextual/logger.service';
+
+/**
+ * Returns a HTMLElement containing the element to copy.
+ * @param {string} html The hmtl to place in angular-html-bind element.
+ * @returns {HTMLElement} The element created by param html.
+ */
+const generateContent = (html: string): HTMLElement => {
+  const container = document.createElement('template');
+  container.innerHTML = `<angular-html-bind>${html.trim()}</angular-html-bind>`;
+  return <HTMLElement>(container.content.firstChild.firstChild);
+};
+
+
+describe('Ck editor copy content service', () => {
+  let loggerService = new LoggerService();
+  let htmlEscaperService = new HtmlEscaperService(loggerService);
+
+  let service: CkEditorCopyContentService;
+  let ckEditorStub: Partial<CKEDITOR.editor>;
+
+  beforeEach(() => {
+    ckEditorStub = {
+      id: 'editor1',
+      insertHtml: (_: string) => {},
+      execCommand: (_: string): boolean => true,
+    };
+    service = new CkEditorCopyContentService(htmlEscaperService);
+  });
+
+  it('not broadcast copy if copy mode is not active', () => {
```

Comment:
`it` block message should start with `should`  (In general describe_message + it_message should be a meaningful sentence.)

Reply:
Done